bin/generate_dist

```diff
@@ -15,6 +15,16 @@ end
 if @cli.options.locales.map(&:downcase).include?("all")
   @cli.options.locales = @cli.glob("data/localizations/categories/*.yml").map { File.basename(_1, ".yml") }
 end
+mapping_groups = MappingRule.all.group_by { |record| [record.input_version, record.output_version] }
```

Comment:
Let's give some breathing room 😛 
```suggestion

mapping_groups = MappingRule.all.group_by { |record| [record.input_version, record.output_version] }
```

Reply:
Also is this the right location in-file? Should this be closer to where it's used? (maybe it already is 🫣 )

Reply:
I moved the block into `generate_dist_files` method